Base/usr/share/man/man8/useradd.md

```diff
@@ -0,0 +1,44 @@
+## Name
+
+useradd - add a new user to the system password file
+
+## Synopsis
+
+```**sh
+# useradd [options] LOGIN
+```
+
+## Description
+
+This program uses the interface defined `pwd.h` to add a new user to the system
+password file. 
+
+A user can either be a `login` user (that is, they have a home
+directory; someone who actually uses to system) or a `system` user (such
+as the `lookup` user found in `/etc/passwd`). 
+
+By default, the user will be added to the `users` group, which has a gid
+of 100.
+
+This program must be run as root.
+
+## Options
+
+* `-u` _UID_: The User Idenfitier for the new user. If this is not specified, a value starting from `1000` will be specified.
```

Comment:
```suggestion
* `-u` _UID_: The user identifier for the new user. If not specified, an unused UID above `1000` will be auto-generated.
```

Reply:
No capilisation even though it's an acronym?